CHANGELOG.md

```diff
@@ -2,9 +2,10 @@
 All notable changes to this project will be documented in this file.
 
 ---
-## [X.X.X]
+## [0.11.0]
```

Comment:
keep [X.X.X], this information is changed at the release time.

Reply:
Done